ray-operator/config/samples/ray-cluster.complete.yaml

```diff
@@ -116,56 +83,12 @@ spec:
           rayCluster: raycluster-complete # will be injected if missing
           rayNodeType: worker # will be injected if missing
           groupName: small-group # will be injected if missing
-        # annotations for pod
-        annotations:
-          key: value
       spec:
-        initContainers:
-        # the env var $RAY_IP is set by the operator if missing, with the value of the head service name
-        - name: init-myservice
-          image: busybox:1.28
-          # Change the cluster postfix if you don't have a default setting
-          command: ['sh', '-c', "until nslookup $RAY_IP.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for myservice; sleep 2; done"]   
         containers:
         - name: machine-learning # must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc'
-          image: rayproject/ray:1.11.0
+          image: rayproject/ray:1.13.0
           # environment variables to set in the container.Optional.
           # Refer to https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/
-          env:
-          - name:  RAY_DISABLE_DOCKER_CPU_WARNING
-            value: "1"
-          - name: TYPE
-            value: "worker"
-          - name: CPU_REQUEST
-            valueFrom:
-              resourceFieldRef:
-                containerName: machine-learning
-                resource: requests.cpu
-          - name: CPU_LIMITS
-            valueFrom:
-              resourceFieldRef:
-                containerName: machine-learning
-                resource: limits.cpu
-          - name: MEMORY_LIMITS
-            valueFrom:
-              resourceFieldRef:
-                containerName: machine-learning
-                resource: limits.memory
-          - name: MEMORY_REQUESTS
-            valueFrom:
-              resourceFieldRef:
-                containerName: machine-learning
-                resource: requests.memory
-          - name: MY_POD_NAME
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.name
-          - name: MY_POD_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.podIP
-          ports:
-          - containerPort: 80
```

Comment:
@DmitriGekhtman what did this port used to be? Should we add the `metrics` port 8080?

Reply:
I have no idea what this port was doing here.

Reply:
I don't believe that the operator interacts with ports specified in workerGroupSpecs.